index.html

```diff
@@ -0,0 +1,151 @@
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <span class="htcj">HTML</span> 
+                    <span class="htcj">CSS</span>
+                    <span class="htcj">Javascript</span><br><br>
+                      <div class="btn3">
+                        <a href="image3.png" target="_blank">See Project</a>
+                      </div>
+            </div>    
+           </div>
+           <div class="container2">
+            <div class="imgdiv2">
+                <img src="image4.png" alt="Nature" height="220px" width="295px">
+            </div>
+                   <div class="multi">
+                    <h1 class="popheading">Multi Post Series</h1>
+                    <p class="popp13">CANOPY <img class="dot" src="dot.png" alt="dot"> 
+                        <span class="popp13 col">Back End Dev</span> <img class="dot" src="dot.png" alt="dot"> 
+                         <span class="popp13 col">2015</span></p>
+                    <br>
+                    <p class="popp15">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <br>
+                    <span class="htcj2">HTML</span> 
+                    <span class="htcj2">CSS</span>
+                    <span class="htcj2">Javascript</span><br><br>
+                      <div class="btn4">
+                        <a href="image4.png" target="_blank">See Project</a>
+                      </div> 
+                      </div>
+                    
+            </div>    
+           </div>
+           <div class="container5">
+            <div class="about">
+                <h1 class="abouth">About Myself</h1>
+                <p class="popp15">Hello I’m a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what you see and
+                     have a project you need coded, don’t hestiate to contact me.</p>
+                 <span class="popp15">Let's Connect</span>   
+                 <section>
+                    <img class="icons" src="github.svg" alt="github" width="17.99px" height="20px">
+                    <img class="icons" src="linkedinicon.svg" alt="linkedin" height="20px" width="22.99px" height="20px">
+                    <img class="icons" src="victory.svg" alt="victory" width="17.99px" height="20px">
+                    <img class="icons" src="twitter.svg" alt="twitter" width="17.99px" height="20px">
+                    <img class="icons" src="m.svg" alt="mimage" width="17.99px" height="20px">
+                </section><br><br>
```

Comment:
- [ ] The list of project technologies icons must be a list `ul` and each of them a `li`. SO kindly remove the section tags surrounding them and replace it with a `ul` and close each icon in an `li` tag :100: 
- UL and LI tags are used to create unordered lists in HTML. The UL tag defines an unordered list and the LI tag is used to define a list item within that unordered list. This allows for a clear and structured way to organize and display a list of items. Additionally, using these tags can make it easier to style the list using CSS :wink:

Reply:
Suggested changes made removed section and added UL tag and displaying images in LI tag. Sending for your review again kindly share your valuable review.